src/quo2/components/wallet/transaction_summary/view.cljs

```diff
@@ -0,0 +1,118 @@
+(ns quo2.components.wallet.transaction-summary.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.wallet.transaction-summary.style :as style]
+            [react-native.core :as rn]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.icon :as icon]
+            [quo2.components.tags.context-tag.view :as context-tag]
+            [utils.i18n :as i18n]))
+
+(def transaction-translation
+  {:send   [i18n/label :t/send]
+   :swap   [i18n/label :t/swap]
+   :bridge [i18n/label :t/bridge]})
+
+(def transaction-icon
+  {:send   :i/send
+   :swap   :i/swap
+   :bridge :i/bridge})
+
+(defn transaction-header
+  [{:keys [transaction
+           theme]
+    :or   {transaction :receive}}]
+  [rn/view
+   {:style style/transaction-header-container}
+   [rn/view {:style style/icon-container}
+    [icon/icon (transaction-icon transaction)
+     {:color (style/icon-color theme)}]]
+   [text/text
+    {:weight :semi-bold
+     :size   :paragraph-1
+     :style  (style/transaction-header theme)}
+    (transaction transaction-translation)]])
+
+(defn prop-text
+  [label theme]
+  [text/text
+   {:weight :regular
+    :size   :paragraph-2
+    :style  (style/prop-text theme)}
+   [i18n/label label]])
```

Comment:
`[i18n/label label]`

`i18n/label` is not a reagent component, it should be called using `()` instead